[2021-02-22 16:04:53,092] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:05:03,945] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:05:14,615] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:05:23,887] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:07:20,476] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:10:38,786] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:15:16,089] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:16:21,387] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:16:32,605] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:16:42,812] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:16:53,704] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:29:48,865] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:30:00,208] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:30:10,441] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:30:17,662] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:32:49,824] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:33:01,157] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:33:11,147] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 16:33:22,070] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_001
[2021-02-22 17:01:59,670] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_001
[2021-02-22 17:02:11,027] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_001
[2021-02-22 17:02:21,301] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_001
[2021-02-22 17:02:32,304] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_001
[2021-02-22 17:02:42,750] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_002
[2021-02-22 17:02:52,883] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_002
[2021-02-22 17:03:03,100] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_002
[2021-02-22 17:03:13,507] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_003
[2021-02-22 17:03:23,694] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_003
[2021-02-22 17:03:33,955] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_003
[2021-02-22 17:07:07,306] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_003
[2021-02-22 17:07:19,754] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_003
[2021-02-22 17:07:30,846] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_003
[2021-02-22 17:17:16,731] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_003
[2021-02-22 17:17:28,106] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_003
[2021-02-22 17:17:38,325] - Logger.py[line:45] - fuc:__console- INFO: 开始执行用例test_formula_abnormal_003
